Pipfile.lock

```diff
@@ -331,6 +321,14 @@
             ],
             "version": "==0.6.1"
         },
+        "more-itertools": {
```

Comment:
I am assuming this was pulled in as a dep from another package and not something we are using directly.

Reply:
Yes, the .lock file is generated by pipenv, nothing manually added.  I did this update with a clean venv that had only the basic pre-installed venv libs in it, like wheel, etc.